index.js

```diff
@@ -393,190 +393,203 @@
           }
         };
 
+        /* DEPRECATED */
         this._createScriptTag = function () {
-          if (!accounts || accounts.length < 1) {
-            that._log('warn', 'No account id set to create script tag');
-            return;
-          }
-          if (accounts.length > 1) {
-            that._log('warn', 'Multiple trackers are not supported with ga.js. Using first tracker only');
-            accounts = accounts.slice(0, 1);
-          }
+          that._registerScriptTags();
+          that._registerTrackers();
+        };
+
+        /* DEPRECATED */
+        this._createAnalyticsScriptTag = function () {
+          that._registerScriptTags();
+          that._registerTrackers();
+        };
+
+        this._registerScriptTags = function () {
+          var document = $document[0],
+              protocol = '',
+              scriptSource;
 
           if (created === true) {
-            that._log('warn', 'ga.js or analytics.js script tag already created');
+            that._log('warn', 'Script tags already created');
             return;
           }
 
           if (disableAnalytics === true) {
-            that._log('info', 'Analytics disabled: ' + accounts[0].tracker);
-            $window['ga-disable-' + accounts[0].tracker] = true;
+            accounts.forEach(function (trackerObj) {
+              that._log('info', 'Analytics disabled: ' + trackerObj.tracker);
+              $window['ga-disable-' + trackerObj.tracker] = true;
+            });
           }
 
-          _gaq('_setAccount', accounts[0].tracker);
-          if(domainName) {
-            _gaq('_setDomainName', domainName);
-          }
-          if (enhancedLinkAttribution) {
-            _gaq('_require', 'inpage_linkid', '//www.google-analytics.com/plugins/ga/inpage_linkid.js');
-          }
-          if (trackRoutes && !ignoreFirstPageLoad) {
-            if (removeRegExp) {
-              _gaq('_trackPageview', getUrl());
+          //
+          // Universal Analytics
+          //
+          if (analyticsJS === true) {
+            protocol = hybridMobileSupport === true ? 'https:' : '';
```

Comment:
Ok. So it is based on the browser choosing its current protocol automatically. May I suggest something explicit like:

````js
var protocol = document.location.protocol;
if (hybridMobileSupport === true) {
  // Enforce https in mobile mode
  protocol = 'https:';
}
````